aslam_offline_calibration/kalibr2/include/kalibr2/CameraModels.hpp

```diff
@@ -0,0 +1,146 @@
+#pragma once
+
+#include <aslam/cameras.hpp>
+#include <aslam/Frame.hpp>
+#include <aslam/backend/ReprojectionError.hpp>
+#include <aslam/backend/SimpleReprojectionError.hpp>
+#include <aslam/backend/CameraDesignVariable.hpp>
+
+
+// Definition of useful structs for camera models in Kalibr2.
+// The following structs define, for each camera model used in Kalibr2 calibration:
+// - Geometry: the camera geometry type
+// - ReprojectionError: the reprojection error type for optimization
+// - Frame: the frame type associated with the geometry
+// - ReprojectionErrorSimple: a simplified reprojection error type
+// - DesignVariable: the design variable for optimization
+// - Distortion: the distortion model used
+// - Shutter: the shutter type (global or rolling)
+// - ProjectionType: the projection model used
+
+namespace kalibr2 {
+
+namespace models {
+
+struct Omni {
+    typedef aslam::cameras::OmniCameraGeometry Geometry;
```

Comment:
@frneer nit: in modern C++, you can also do:
```suggestion
    using Geometry = aslam::cameras::OmniCameraGeometry;
```
which is a bit more idiomatic than old `typedef` (though in this specific case they are functionally equivalent).

Reply:
Done.